pkg/volume/csi/csi_attacher_test.go

```diff
@@ -1216,7 +1217,10 @@ func TestAttacherMountDevice(t *testing.T) {
 			createAttachment:        true,
 			populateDeviceMountPath: true,
 			shouldFail:              true,
-			spec:                    volume.NewSpecFromPersistentVolume(makeTestPV(pvName, 10, testDriver, "test-vol1"), true),
+			// NOTE: We're skipping this test on Windows because os.Chmod is not working as intended, which means that
```

Comment:
is this temporary or "it doesn't work that way" ?

Reply:
Windows permissions work a bit differently, and are a bit more fine grained. However, the Windows permissions could be boiled down to something Linux-like. I do have a PR for something like this (https://github.com/kubernetes/kubernetes/pull/104660), and we could revisit this particular skip if that gets merged.

Reply:
So are we OK to merge this without Windows supporting it?

@msau42 @jingxu97

Reply:
I think it is ok to add a TODO and revisit here after windows permission support PR is merged.

Also want to confirm if in test we do not use Chmod to change file permission, the test will fail for Linux?

Reply:
Added TODO. Also, the test will fail on Linux without the Chmods:

```
=== RUN   TestAttacherMountDevice/failure_PV_with_existing_data
    csi_attacher_test.go:1294: Running test case: failure PV with existing data
    csi_client_test.go:280: calling fake.NodeGetCapabilities for NodeSupportsStageUnstage...
    csi_client_test.go:227: calling fake.NodeStageVolume...
    csi_attacher_test.go:1369: test should fail, but no error occurred
```